autosklearn/pipeline/components/data_preprocessing/text_encoding/bag_of_word_encoding.py

```diff
@@ -79,6 +83,8 @@ def get_properties(dataset_properties: Optional[DATASET_PROPERTIES_TYPE] = None
     def get_hyperparameter_search_space(dataset_properties: Optional[DATASET_PROPERTIES_TYPE] = None
                                         ) -> ConfigurationSpace:
         cs = ConfigurationSpace()
+        hp_ngram_range = CSH.UniformIntegerHyperparameter(name="ngram_range", lower=1, upper=3,
```

Comment:
Hey, should this maybe be `ngram_range_upper_bound`?

Reply:
Thinking about this again, maybe we should not do this now because it will again change all files that contain meta-data -> maybe put this on a TODO for V2?